### PR TITLE
added menu items

### PR DESCRIPTION
**Pull request by rajgog**

### CODE REVIEW

**Review by rajgog — COMMENTED** [review]
approved

**Comment by rajgog** [comment]
reopened

File: src/webapp/app/Client.Application/Layout/MainLayout.razor [hunk 1]
```diff
@@ -19,9 +19,22 @@
             <div class="profile">
                 <AuthorizeView>
                     <Authorized>
-                        <FluentButton BackgroundColor="var(--neutral-layer-4)"  @onclick="BeginLogOut" Title="Profile">
-                            <FluentEmoji Value="@(new Emojis.AnimalsNature.Flat.Default.Lion())" Width="35px" />
+                        <FluentButton id="btnOpenProfile" BackgroundColor="var(--neutral-layer-4)"  @onclick="() => _visibleL = !_visibleL" Title="Profile">
+                            <FluentPersona Name="@context.User.Identity?.Name"
```
Comment (Ayon-Neo) [inline src/webapp/app/Client.Application/Layout/MainLayout.razor]: Please check for null value effect on the frontend

Comment (rajgog) [reply src/webapp/app/Client.Application/Layout/MainLayout.razor]: Added check to handle null

**Review by rajgog — COMMENTED** [review]
Screenshot 2024-01-23 at 9 28 52 PM

**Pull request merged**